src/AutorestSwift/View Models/ConstantViewModel.swift

```diff
@@ -35,13 +35,17 @@ struct ConstantViewModel {
     let comment: ViewModelComment
     // default value of the proeprty
     let defaultValue: ViewModelDefault
+    let required: Bool
+    let serializedName: String
 
     /// Initialize from Value type (such as Property or Parameter)
-    init(from schema: ConstantSchema) {
+    init(from schema: ConstantSchema, required: Bool) {
         let name = (schema.serializedName ?? schema.name).lowercasedFirst
         assert(!name.isEmpty)
         self.name = name
+        self.serializedName = (schema.serializedName ?? schema.name)
         self.comment = ViewModelComment(from: schema.description)
         self.defaultValue = ViewModelDefault(from: schema.value.value, isString: true, isOptional: false)
+        self.required = required
```

Comment:
What is the use case of an "optional constant"?  A nil constant is pointless, and by definition a constant can't change, so how does this even work?